build/bin/build-server-rpm

```diff
@@ -250,7 +251,7 @@ d }\" \
             sudo chown -R $(id -u):$(id -g) /home/builder/rpm/RPMS /home/builder/rpm/SOURCES
         "
 
-
+        
```

Comment:
@adivinho, please configure your editor to strip trailing whitespace. That's not the first time.